content/ember-6-0-released.md

```diff
@@ -0,0 +1,231 @@
+
+---
+
+## EmberData
+
+EmberData is the official data persistence library for Ember.js applications.
+
+EmberData 5.12 will remain an actively supported LTS for the entire duration of the 5.x cycle, extending until the first 6.x LTS is released. This is in addition to our regular LTS support policy, and only applies to the EmberData project.
+
+This support extends beyond bug-fixes. If minor enhancements can be made to better support new presentation class implementations that support the 5.x series we will willingly accept them. Our goal is that we want no one left behind.
+
+To learn about the motivation and goals for upcoming changes to EmberData in 5.x,
+read the [blog post, EmberData 5.X Update](https://blog.emberjs.com/ember-data-5-x-update-2023-04-15/).
+
+<!-- alex ignore retext-equality -->
+
+This will help you form the mental model of what to expect across the 5.x series,
+and understand deprecation removals in the context of the upcoming goals.
+
+The sections below will provide a high-level summary of what has changed between
+EmberData 5.12 and 6.0.
+
+### Changes in EmberData
+
+#### Bug Fixes
+
+EmberData is the official data persistence library for Ember.js applications.
+
+This release cycle EmberData is again electing to not release a new minor.
+
+Work in the library continues to be focused primarily on aligning 4.12 and 5.3 to enable as many folks in the community to seamlessly upgrade as possible.
+
+<!-- alex ignore easy -->
+
+We continue to take our time with 5.4 as we intend to introduce a major new paradigm with it (the replacement for @ember-data/model) and believe releasing this new feature at once on the heels of two very robust LTSs (4.12 and 5.3) is the best approach for users looking to navigate upgrades easily.
+
+Please see the [blog post](https://blog.emberjs.com/updates-to-ember-data-versioning-strategy) addressing EmberData's new versioning strategy for more information.
+
+---
+
+## Ember CLI
+
+Ember CLI is the command line interface for managing and packaging Ember.js applications.
+
+### Upgrading Ember CLI
+
+You may upgrade Ember CLI using the `ember-cli-update` project:
+
+```bash
+npx ember-cli-update
+```
+
+This utility will help you to update your app or addon to the latest Ember CLI version. You will probably encounter merge conflicts, in which the default behavior is to let you resolve conflicts on your own. For more information on the `ember-cli-update` project, see [the GitHub README](https://github.com/ember-cli/ember-cli-update).
+
+While it is recommended to keep Ember CLI versions in sync with Ember and EmberData, this is not required. After updating ember-cli, you can keep your current version(s) of Ember or EmberData by editing `package.json` to revert the changes to the lines containing `ember-source` and `ember-data`.
+
+### Changes in Ember CLI 6.0
```

Comment:
Do you have anything you'd want to call out here that happened in the 5.x series for ember-cli or the long tail that includes? cc @ef4 @NullVoxPopuli @bertdeblock

Reply:
Having the official typescript types in the blueprint could get called out here.

Also, the blueprint getting updated to the newer flat-style eslint config.

If we're looking for things during all of 5.x, automatically choosing a free port happened in 5.4 and is a pleasant quality-of-life improvement.

Reply:
Flat-style eslint config is in 6.1